tests/stdlib/tos.nim

```diff
@@ -88,6 +90,17 @@ echo "Dirs:"
 for path in walkDirs(dname/"*"):
   echo path.norm
 
+# Test that the `recursive = false` argument doesn't remove non-empty dirs or files
+try:
+  removeDir(dname, recursive = false)
+except OSError:
+  echo "RaisesNonEmptyDir"
+
+try:
+  removeDir(dname/files[0], recursive = false)
+except OSError:
+  echo "RaisesFile"
```

Comment:
```diff
-try:
-  removeDir(dname/files[0], recursive = false)
-except OSError:
-  echo "RaisesFile"
```

I think this test can be removed, or have `recursive` omitted to deassociate it from the `recursive` parameter.

Reply:
Ah yes, this test was a leftover from the sleep-deprivation.